src/lib.rs

```diff
@@ -292,7 +297,7 @@ impl<T: Counter> Histogram<T> {
     /// Find the bucket the given value should be placed in.
     ///
     /// May panic if the given value falls outside the current range of the histogram.
```

Comment:
Not removing this comment until I have more tests and better understanding of what happens when out-of-range values are used. Values that are too small will just get shifted until they're 0, so that's safe at least.